habitat-lab/habitat/sims/habitat_simulator/object_state_machine.py

```diff
@@ -127,6 +130,7 @@ def update_state(
 
     def default_value(self) -> Any:
         """
+        BUG: IS THIS FUNCTION SAFE??
```

Comment:
I don't think this is a bug. Default return value when none is provided is None, so this will return that.
Child classes will return other values as needed.